magicctapipe/scripts/lst1_magic/README.md

```diff
@@ -31,6 +31,3 @@ MAGIC+LST-1 analysis starts from MAGIC calibrated data (\_Y\_ files), LST-1 DL1
 - `lst1_magic_create_irf.py` to create the IRF
 - `lst1_magic_dl2_to_dl3.py` to create DL3 files, and `create_dl3_index_files.py` to create DL3 HDU and index files
 
-## High level analysis
```

Comment:
why this part is removed from the readme?

Reply:
It is back now. This part of the README will be updated in the next PR.